src/Resources/views/Navigation/navigationBlocks.html.twig

```diff
@@ -2,29 +2,43 @@
 {# level int #}
 {# maxLevels int #}
 {# expandedLevels int #}
+{% block prefix %}{% endblock %}
+{% block prefix_separator %}-{% endblock %}
+{%- block prefix_final -%}
+    {{ block('prefix') is not empty ? block('prefix') ~ block('prefix_separator') }}
+{%- endblock -%}
+
 {% block navigation_if_visible %}
+    {%- set navigation_base_class = block('prefix_final') ~ 'nav' -%}
+
     {% if level < maxLevels and (level < expandedLevels or root.activePath) %}
         {% set visibleNodes = root.children | filter(node => node.visible) %}
         {% if visibleNodes is not empty %}
-            {{ block('navigation_list') }}
+            <nav class="{{ navigation_base_class }} {% block navigation_additional_class %}{% endblock %}">
```

Comment:
Ist glaube ich ein weiterer BC-Break, weil mit Version 4 erwartet wurde, das `<nav>` im Projekt zu schreiben (weiß nicht warum, gibts da mehr Flexpoints?). Wäre dann nach dem Upgrade ein doppeltes `<nav>` und man müsste nicht nur CSS anpassen.

Reply:
Sehe das wie Malte. Wenn wir die `<nav>` Elemente mit reinnehmen wollen, brauchen wir noch einen weiteren flexpoint für `aria-label` um die Navigation Landmarks in AT unterscheidbar zu machen ("main", "breadcrumbs", "legal" sind typische englische Bezeichnungen, für die ich keine prägnanten deutschen Übersetzungen parat habe).

Reply:
Guter Punkt